pl_examples/domain_templates/computer_vision_fine_tuning.py

```diff
@@ -55,7 +55,7 @@
 import pytorch_lightning as pl
 from pl_examples import cli_lightning_logo
 from pytorch_lightning import _logger as log
-from pytorch_lightning import LightningDataModule
+from pytorch_lightning import LightningDataModule, seed_everything
```

Comment:
all changes in this file should be irrelevant I think.

Reply:
Zach is using this script for making sure TPU works. Just added everything to make it compatible with TPU training.